packages/opentelemetry-types/README.md

```diff
@@ -0,0 +1,11 @@
+# `opentelemetry-types`
+
+> TODO: description
+
+## Usage
+
+```
+const opentelemetryTypes = require('opentelemetry-types');
```

Comment:
I thought this package only contained the TypeScript definitions?

Reply:
Updated the README file, the default content was auto-generated by some extension.